### ISSUE TITLE
Typescript issue with Cloudflare Pages

### ISSUE THREAD

**Issue by isaac-mcfadyen** [BuilderIO/qwik#541]
### Qwik Version

0.0.21-0

### Operating System (or Browser)

MacOS

### Node Version (if applicable)

v18.2.0

### Which component is affected?

Qwik Rollup / Vite plugin

### Expected Behaviour

- Qwik plugin registers with Vite with no issues.
- No errors from IDE.

### Actual Behaviour

- Plugin runs fine.
- However, when using Cloudflare Pages, the plugin registration looks like this:
```js
qwikVite({
    ssr: {
        outFile: join(__dirname, "functions", "[[path.js]]"),
    },
}),
```
- Because the types for `qwikVite` only include an outDir, not an outFile, there's a red line and VSCode gives errors about type mismatches.

### Additional Information

- Cosmetic, doesn't impact running of application.

**Comment by manucorporat** [comment]
should be fixed in latest!

**Comment by isaac-mcfadyen** [comment]
Fixed in 0.0.21! 🎉
